### ISSUE TITLE
it may have bug in uncropping

### ISSUE THREAD

**Issue by Shawn-Yu-1** [Janspiry/Palette-Image-to-Image-Diffusion-Models#56]
I trained the uncropping model on a subset of palces2 that have 9000 pics of image. I found the metric loss is going bigger as I trained.
but I haven't finished the training yet, but I think there may some error in the code. may be, it has any data augmentation, may lead some overfit.

**Comment by Shawn-Yu-1** [comment]
I have seen the result of the uncropping in the README, I think it may just run for enough data. I see you run about 80 million pics  of image , that is two expensive for train a model. BUT think you for the code.

**Comment by Janspiry** [comment]
Intuitively its loss will go down, or should you adjust some learning rate?
